config/webpack.plugins.js

```diff
@@ -87,6 +88,19 @@ const plugins = (dev = false, beta = false, restricted = false) => {
       __SENTRY_DEBUG__: false,
     }),
     ...(dev ? [new ReactRefreshWebpackPlugin()] : []),
+    // Put the Sentry Webpack plugin after all other plugins
+    process.env.SENTRY_AUTH_TOKEN &&
```

Comment:
Adding a plugin conditionally like this will append `false` or `undefined` to the array. The condition should also verify that all required variables exist in the environment.